reference-architecture/aws-ansible/ansible.cfg

```diff
@@ -0,0 +1,22 @@
+# config file for ansible -- http://ansible.com/
+# ==============================================
+[defaults]
+#callback_plugins = ../openshift-ansible/ansible-profile/callback_plugins
+forks = 50
+host_key_checking = False
+hostfile = inventory/aws/hosts/ec2.py
+remote_user = ec2-user
+gathering = smart
+fact_caching = jsonfile
+fact_caching_connection = .ansible/cached_facts
+fact_caching_timeout = 900
```

Comment:
I would suggest against using fact caching, unless it will only be used through the wrapper script and the wrapper clears the fact cache between runs.

Only reason I say that is because the fact caching will cause unintuitive behavior for users that aren't familiar with the feature in advance.